src/configuration/content-evaluator-config.ts

```diff
@@ -16,22 +16,6 @@ export const contentEvaluatorConfigurationType = Type.Object({
           select: ["ISSUE_SPECIFICATION"],
           relevance: 1,
         },
-        {
```

Comment:
Why are these removed?

Reply:
I have removed the default config options to apply fixed relevance to PR comments here. Because PR comments doesn't need a fixed relevance anymore. We are now sending them to OpenAI to measure their relevance value. Fixed relevance has higher priority than OpenAI relevance. If specified here for any comment type, the comment will not be sent to OpenAI, and fixed relevance will be applied.

Reply:
If they are added back to the configuration via the file, will they override anything? If these options are not available at all anymore, it would be great to remove them from the codebase too, otherwise ignore my comment.

Reply:
These options are available. They are just not needed in the default config. 

If you add it to config, they have the highest priority and they will override OpenAI evaluation for the specified type of comment.